dist/systemd/obsapidelayed-sphinx.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description = OBS Sphinx server
```

Comment:
I would suggest: `Open Build Service Sphinx Search Daemon`. And name the unit `obs-sphinx`

Reply:
@hennevogel 

> I would suggest: `Open Build Service Sphinx Search Daemon`. And name the unit `obs-sphinx`

Changed.